locallibrary/catalog/admin.py

```diff
@@ -1,3 +1,47 @@
 from django.contrib import admin
 
 # Register your models here.
+from .models import Author, Genre, Book, BookInstance
+
+# admin.site.register(Book)
+# admin.site.register(Author)
+admin.site.register(Genre)
+
+
+# admin.site.register(BookInstance)
+
+# Define the admin class
+class AuthorAdmin(admin.ModelAdmin):
+    list_display = ('last_name', 'first_name', 'date_of_birth', 'date_of_death')
```

Comment:
Trả lời câu hỏi : Khai báo sau có ý nghĩa gì ?
- list_display
- fields
- fieldsets
- list_filter

Reply:
-list_display là một option trong ModelAdmin dùng để xác định xem trường nào sẽ được hiển thị ở changelist của admin. ở trong hình trên thì sẽ là hiển thị 4 cột gồm lastname, firstname, date of birth và date of death.

-fields dùng để hiển thị các trường trên form theo thứ tự. Fields mặc định sẽ được hiển thị theo chiều dọc, nhưng có thể hiển thị theo chiều ngang nếu group theo kiểu tuple.

fieldsets dùng để điều khiển layout của trang add/change của admin.

list_filter dùng để tạo 1 filter bên phải màn hình change list. Tùy thuộc kiểu dữ liệu thì django có thể support 1 sẵn 1 số filter (vd như thời gian,. . .).